sphinx/source/docs/user_guide/annotations.rst

```diff
@@ -194,7 +193,7 @@ arrow shaft:
     my_arrow.line_color = "blue"
     my_arrow.line_alpha = 0.6
 
-Arrows may also be configured to refer to additional non-default x- or
+Arrows may also be configured to refer to additional non-default x-ranges or
```

Comment:
Do our new guidelines actually suggest such change? The original seems to me more readable.

Reply:
cc @tcmetzger

Reply:
[Google's developer style guide](https://developers.google.com/style/hyphens#suspended-hyphens) (which we use as the basis for our guidelines) does allow you to leave out the base for all except the last modifier in a situation like this (https://developers.google.com/style/hyphens#suspended-hyphens). So I would keep the original version here.

Reply:
Sure noted. Will make the necessary changes.